### DISCUSSION TITLE
For the "Edit call service node"

### DISCUSSION THREAD

**Discussion by fixx-er** [hassio-addons/addon-node-red#1678] [Q&A]
Choosing > Domain > scene
then Service "Apply"
the Area, device, and entity disappear.
Is this normal?

**Comment by frenck** [comment]
I dunno, your question is about the Home Assistant module/node for Node-RED, not the add-on. Thus you are not in the right place.

../Frenck

**Reply by ndoggac** [reply]
> I dunno, your question is about the Home Assistant module/node for Node-RED, not the add-on. Thus you are not in the right place.
> 
> ../Frenck

For the ignorant (read: me), where is the proper location so I can submit the proper bug report?  It's been listed (apparently improperly) as an issue here for nearly two months with no analogous reply.  Wondering if a similar issue has been submitted in the proper location.  Just looking to get it fixed, as it sees to be affecting a lot of people.  Thanks.

**Comment by Tanjohnson99** [comment]
I have the same issue.

**Reply by frenck** [reply]
Then I have the same answer... 🤷‍♂️
